### DISCUSSION TITLE
How to tell DR not to convert images?

### DISCUSSION THREAD

**Discussion by rado84-github** [darkreader/darkreader#9948] [Q&A]
On this website, if DR is enabled, it converts the image of the article which makes it look like a photo film from the 90's:
https://www.kaldata.com/it-%D0%BD%D0%BE%D0%B2%D0%B8%D0%BD%D0%B8/%D0%B8%D0%B7%D0%BB%D0%B5%D0%B7%D0%B5-%D0%B3%D0%BE%D0%BB%D1%8F%D0%BC%D0%BE-%D0%BE%D0%B1%D0%BD%D0%BE%D0%B2%D1%8F%D0%B2%D0%B0%D0%BD%D0%B5-%D0%B7%D0%B0-%D1%81%D0%BF%D0%B5%D1%86%D0%B8%D0%B0%D0%BB%D0%B8-392439.html

![Picture_created_28-09-2022_16-13-20](https://user-images.githubusercontent.com/41172201/192787824-cde07d75-f18b-40b6-a469-77b95f4053fe.png)

So, is there any way to tell DR not to convert these images? I tried adding a "disable for this site" address like this: kaldata.com/wp-content/uploads/* but it didn't work and DR still converted the image.

I saw this issue, but none of the mentioned there was able to solve my problem:
https://github.com/darkreader/darkreader/issues/886

**Comment by Gusted** [comment]
That's an issue, you should report it instead of trying to work around it. Will be fixed by #9959
